internal/io/pending_command.go

```diff
@@ -59,7 +59,7 @@ func (p Pending) View() string {
 	if !p.quitting {
 		return fmt.Sprintf("%s %s", p.spinner.View(), p.output)
 	}
-	return p.output + "\n"
```

Comment:
I don't understand why at the moment but by not printing a newline at the end, upon the final output before the program exits, the last line is swallowed. I think it might be best to check if the last character in `output` is a newline and add one if not